<?xml version="1.0"?>
<launch>
  <include file="$(find target_reaching_nengo)/launch/main_TR_CL_nengo_ur5e.launch" >
    <arg name="is_multi_robot_experiment" value="True"/>
    <arg name="gazebo_target_base_y" default="2.4" />
  </include>
</launch>
